ui/src/app/pages/data/concept/concept-add-modal.tsx

```diff
@@ -96,13 +109,39 @@ export const ConceptAddModal = withCurrentWorkspace()
     }
   }
 
+  isSurvey() {
```

Comment:
Nit: Make this a get function? (so you can call as parameter rather than method)